spec/helper/spec_helper.rb

```diff
@@ -0,0 +1,113 @@
+require 'rubygems'
+require_relative 'coveralls_fix'
+require_relative 'trace_helper'
+require_relative '../../lib/hockey_gerrit'
+
+require 'webmock/rspec'
+require 'pry'
+
+module SpecHelper
+  def path_for(path)
+    File.expand_path(File.join(__dir__, '..', 'fixtures', path))
+  end
+
+  module_function :path_for
+
+  def ipa_path
+    @ipa_path ||= path_for 'fake.ipa'
+  end
+
+  def dysm_path
+    @dysm_path ||= path_for 'fake.app.dSYM.zip'
+  end
+
+  def fake_broken_path
+    @fake_broken_path ||= path_for 'fake_broken'
+  end
+
+  def upload_request_path
+    @expected_request_path ||= path_for 'upload_request.yml'
+  end
+
+  module_function :upload_request_path
+
+  def self.disable_net
+    File.exist?(upload_request_path)
+  end
+
+  def post_url # used by webmock
+    'https://upload.hockeyapp.net/api/2/apps/upload'
+  end
+
+  def stub_valid_obj
+    config_url = '{ "config_url": "https://upload.hockeyapp.net/manage/apps/123456/app_versions/9"}'
```

Comment:
Tests depend on this url (with upload changed to rink). Would it be better to declare 'upload_url' and 'rink_url' variables so we are not using strings in the tests?